packages/FormElement/stories/examples/DatePicker.js

```diff
@@ -1,21 +1,32 @@
 import React from "react";
 import DatePicker from "@paprika/date-picker";
+import L10n from "@paprika/l10n";
 import FormElement from "../../src";
 
-const { Content, Label } = FormElement;
+const { Label, Content, Instructions } = FormElement;
 
 export default function DatePickerExample() {
-  const errorText = "";
+  const [date, setDate] = React.useState();
+
+  function handleChange(newDate) {
+    setDate(newDate);
+  }
+
+  const dateFormat = "YYYY/MM/DD";
+
   return (
-    <FormElement>
-      <Label>Form Label</Label>
-      <Content>
-        {a11yProps => (
-          <DatePicker onError={() => {}} hasError={Boolean(errorText.length)} onChange={() => {}}>
-            <DatePicker.Input {...a11yProps} />
-          </DatePicker>
-        )}
-      </Content>
-    </FormElement>
+    <L10n>
+      <FormElement>
+        <Label>Start Date</Label>
+        <Instructions>Enter date in {dateFormat} format.</Instructions>
+        <Content>
+          {a11yProps => (
+            <DatePicker date={date} onChange={handleChange}>
+              <DatePicker.Input placeholder={dateFormat} dateFormat={dateFormat} {...a11yProps} />
```

Comment:
Oh no it doesn't actually but perhaps we should encourage it to just be added to the root component? Or should it be just an inputProp?

Reply:
Interesting 🤔 

It works on either, but the one on the `Input` takes precedence.  I think since it's the `Input` that _needs_ it, it makes more sense to remove the prop (and `humanFormat`, etc) from the `<DatePicker>`.  But that's for a different issue/PR.

cc @DanilAgafonov @allison-c